.travis/build-windows.sh

```diff
@@ -7,11 +7,16 @@ mkdir build
 cd build
 
 if [ "$1" == "release" ]; then
+    if [ "$2" == "openmp" ]; then
```

Comment:
Isn't == a bashism? sh is bash on Travis, so it doesn't really matter.

Reply:
Updated it to sh syntax, and explicitly use bash in sign-macos.sh, see latest push